spec/API_specification/data_types.md

```diff
@@ -78,3 +78,29 @@ Implementations may provide other ways to specify data types (e.g.,
 
 A conforming implementation of the array API standard may provide and support additional data types beyond those described in this specification.
 ```
+
+(data-type-categories)=
+## Data Type Categories
+
+For the purposes of this spec, the following data type categories are defined.
+Libraries do not need to organize dtypes according to these categories. These
+are only for organizing the functions in this spec itself. Future versions of
+the spec will include additional categories for complex data types.
+
+### Numeric Data Types
+
+`int8`, `int16`, `int32`, `int64`, `uint8`, `uint16`, `uint32`,
```

Comment:
Should we make this a list (i.e., demarcated by bullets)?

Reply:
I think it's fine as is, clear enough and bullet lists will make this quite long.